lib/vp8_unittest.py

```diff
@@ -70,6 +70,10 @@ def test_ConfigurationFixups(self):
     value_set_out = codec.ConfigurationFixups(value_set)
     self.assertEqual('-5', value_set_out.GetValue('cpu-used'))
 
+  def test_EncoderVersion(self):
+    codec = vp8.Vp8Codec()
+    self.assertRegexpMatches(codec.EncoderVersion(),
+                             r'WebM Project VP8 Encoder')
```

Comment:
The important numbers are to the right - this says "the string returned some information with VP8 in it" (which would actually have caught the error if the function didn't raise, above).
I didn't want to have something that was fragile if the webm project changed its version formatting, but if webm changed the naming convention, it was worth faililng on.